rubicon_ml/intake_rubicon/publish.py

```diff
@@ -18,13 +19,24 @@ def publish(
         and key to log the generated YAML file to. S3 buckets
         must be prepended with 's3://'. Defaults to None,
         which disables writing the generated YAML.
+    base_catalog_filepath : str, optional
+        Similar to output_filepath except this argument is used as a
+        base fileto update exisiting intake catalog. Defaults to None,
```

Comment:
```suggestion
        Similar to `output_filepath` except this argument is used as a
        base file to update exisiting intake catalog. Defaults to None,
```

just nitpicking - use backticks around argument names in the docstrings